themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -346,6 +346,10 @@ fine-oidc-endpoint-conf=Fine Grain OpenID Connect Configuration
 fine-oidc-endpoint-conf.tooltip=Expand this section to configure advanced settings of this client related to OpenID Connect protocol
 access-token-signed-response-alg=Access Token Signature Algorithm
 access-token-signed-response-alg.tooltip=JWA algorithm used for signing access tokens.
+access-token-encrypted-response-alg=Access Token Encryption Key Management Algorithm
+access-token-encrypted-response-alg.tooltip=JWA Algorithm used for key management in encrypting access tokens. This option is needed if you want encrypted access tokens. If left empty, access tokens are just signed, but not encrypted.
+access-token-encrypted-response-enc=Access Token Encryption Content Encryption Algorithm
+access-token-encrypted-response-enc.tooltip=JWA Algorithm used for content encryption in encrypting Access tokens. This option is needed just if you want encrypted access tokens. If left empty, access Tokens are just signed, but not encrypted.
```

Comment:
How about the folowing?

`access-token-encrypted-response-enc.tooltip=JWA Algorithm used for content encryption in encrypting access tokens. This option is needed just if you want encrypted access tokens. If left empty, access tokens are just signed, but not encrypted.`

Access tokens -> access tokens
access Tokens -> access tokens